pkgs/by-name/go/godot_4/package.nix

```diff
@@ -34,13 +36,13 @@
   withTarget ? "editor",
   withTouch ? true,
   withUdev ? true,
+  withWayland ? true,
```

Comment:
It should be false. Current package is for X11. Wayland is the new addition.

Reply:
If you want hydra to continuously build for Wayland you have to add an alias for that. Either in `pkgs/top-level/all-packages.nix` or `pkgs/by-name`. `all-packages.nix` is likely more appropriate since it is just an override.
X11 users should not have to download Wayland.
X11 has always been the default option.